File: dayOne/java/NumberGuessingApp.java
import java.util.Scanner;

public class NumberGuessingApp {
    public static void main(String... args){
    
        Scanner input = new Scanner(System.in);
        
        double random = Math.floor(Math.random() * 100) + 1;
        int attempt = 1;
        
        System.out.println("Welcome to number guessing game.");
        System.out.println("You have 5 attempts.");
        
        while (attempt <= 5){
            System.out.print("Guess a number between 1 - 100: ");
            int guess = input.nextInt();
            
            if (guess == random){
                if (attempt == 1) System.out.print("Legendary");
                else if (attempt == 2) System.out.print("Excellent");
                else if (attempt == 3 || attempt == 4) System.out.print("Good");
                else if (attempt == 5) System.out.print("close");
                System.out.println();
                
                System.out.printf("you used %d attempts \nthe correct answer was %d ", attempt, random);
                break;
            }
            else if (guess < random){
                System.out.println("Your guess is lower. TRY AGAIN");
                System.out.println();
                attempt++;
            }
            else if (guess > random){
                if (guess > 100){
                    System.out.println("Final warning please enter a number from 1 - 100");
                    System.out.println();
                }else{
                    System.out.println("Your guess is higher. TRY AGAIN");
                    System.out.println();
                    attempt++;
                }
                
            }
            
            
        }
    }
}

File: dayOne/python/number_guessing_app.py
import random

random = random.randint(1, 2)

attempt = 1

print("Welcome to number guessing game.")
print()
print("You have 5 attempts.")
print()

while attempt <= 5:
    guess = int(input("Guess a number from 1 - 100: "))
    
    if guess == random:
        if attempt == 1: print("Legendary")
        elif attempt == 2: print("Excellent")
        elif attempt == 3 or attempt == 4: print("Good")
        elif attempt == 5: print("close")
        print()

        print(f"you used {attempt} attempts \nthe correct answer was {random} ") 
        break
        
    elif guess < random:
        print("Your guess is lower. TRY AGAIN")
        print()
        attempt+=1
    
    elif guess > random:
        if guess > 100:
            print("WARNING GUESS A NUMBER FROM 1 - 100")
            print()
        else: 
            print("Your guess is higher. TRY AGAIN")
            print()
            
            attempt+=1
            
    else: print("Better luck")
